chapters/annotations.tex

```diff
@@ -383,20 +383,24 @@ \section{Annotations for Simulations}\label{annotations-for-simulations}
 \end{center}
 
 \begin{annotationdefinition}[experiment]
-% henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
-\begin{synopsis}[grammar]\begin{lstlisting}
-"experiment"
-   [ "(" [ experimentOption { "," experimentOption } ] ")" ]
-
-experimentOption:
-   "StartTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "StopTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "Interval" "=" UNSIGNED-NUMBER
-   | "Tolerance" "=" UNSIGNED-NUMBER
-\end{lstlisting}\end{synopsis}
+
+\begin{synopsis}
+The \fmtannotationindex{experiment} annotation has the following contents:
+\begin{lstlisting}[language=modelica]
+record experiment
+  Real StartTime(unit = "s") = 0;
+  Real StopTime(unit = "s");
+  Real Interval(unit = "s");
+  Real Tolerance;
```

Comment:
Might as well spell out the unit of `Tolerance` (would allow rejecting bad units in case we would allow parameter expressions in the future):
```suggestion
  Real Tolerance(unit = "1");
```